Add SBOM verification to docs (#42996)
kubernetes/website: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

- [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
- [Configuration Options](https://github.com/sigstore/policy-controller/tree/main/config)

## Verify the Software Bill Of Materials (SBOM)

The Kubernetes SBOM is signed as well and can be verified by using the sigstore

> certificate and signature or the corresponding SHA files:

Reviewer: Nit: I think this would improve readability. I hope I got it correctly.
```suggestion
certificate and signature, or the corresponding SHA files:
```

Author: Fixed as suggested.

```shell
VERSION=$(curl -Ls https://dl.k8s.io/release/stable.txt)

# SHA512 verification

> curl -Ls "https://sbom.k8s.io/$VERSION/release" -o "$VERSION.spdx"

Reviewer: I think we should add some text description about what each command is doing.

Author: Yep, done :+1:

- [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
- [Configuration Options](https://github.com/sigstore/policy-controller/tree/main/config)

> ## Verify the Software Bill Of Materials (SBOM)
> 
> The Kubernetes SBOM is signed as well and can be verified by using the sigstore
> certificate and signature, or the corresponding SHA files:

Reviewer 2: ```suggestion
## Verify SBOM

The Software Bill Of Materials (SBOM) for Kubernetes can be verified by using either the sigstore
certificate and signature or the corresponding SHA files:
```
Is it better to keep the heading concise and use a full spelling in the body text?

Reviewer 3: It makes sense to spell out SBOM in the heading. In the sentence, you can spell out SBOM and provide the acronym.

```
## Verify the Software Bill of Materials

The Kubernetes Software Bill of Materials (SBOM) can be verified 

or
You can verify the Kubernetes Software Bill of Materials (SBOM) by using the sigstore certificate ...
```

Author: Changed as suggested, thanks folks! :+1:
